packages/features/landing_page_screen/lib/src/landing_page.dart

```diff
@@ -222,11 +213,10 @@ class _ProjectPreview extends StatelessWidget {
                 openProject();
               }
             },
-            icon: SvgPicture.asset(
-              'assets/svg/ic_edit_circle.svg',
-              height: 264,
-              width: 264,
-            ),
+            icon: const IcSvg(
```

Comment:
Lets make this widgets name more clearer. IconSvg?

Reply:
Done